test/arm/neon/cmla_lane.c

```diff
@@ -65,18 +63,21 @@ test_simde_vcmla_lane_f16 (SIMDE_MUNIT_TEST_ARGS) {
     simde_float16x4_t r;
     SIMDE_CONSTIFY_2_(simde_vcmla_lane_f16, r, (HEDLEY_UNREACHABLE(), simde_vdup_n_f16(SIMDE_FLOAT16_VALUE(0.0))), test_vec[i].lane, r_, a, b);
```

Comment:
Alas MSVC has a long-standing problem with the `SIMDE_CONSTFY_` macros in the tests, you will need to un-roll this manually. See https://github.com/simd-everywhere/simde/blob/493614901bcecea23554b52a3a55fc0772c2de8b/test/x86/clmul.c#L124-L139 for an example